pangocairo/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 // Licensed under the MIT license, see the LICENSE file or <https://opensource.org/licenses/MIT>
 
 #![allow(deprecated)]
+#![cfg_attr(feature = "dox", feature(doc_cfg))]
```

Comment:
This only works with nightly or not?

Reply:
Only for nightly, which is why I added this line.

Reply:
If someone wants to build docs locally that's a bit annoying, but I guess acceptable :)